dumper/lib-dumper-spi/src/main/java/com/google/edwmigration/dumper/plugin/lib/dumper/spi/OracleMetadataDumpFormat.java

```diff
@@ -687,7 +687,9 @@ interface XmlSequences {
     final String ZIP_ENTRY_NAME_ALL = "ALL.Sequences-XML.csv";
 
     enum Header {
+      @Deprecated // `OWNER` is not associated with any column. Please use 'SEQUENCE_OWNER' instead.
```

Comment:
Previous PR was not released yet, so I think you can simply remove `OWNER` column as it is invalid.

Reply:
It wasn't added as part of the previous PR 500. `XmlSequences` with 2 columns + XML existed long before that.